docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md

```diff
@@ -0,0 +1,49 @@
+# Supermicro X11 LGA1151 Series theory of operation
+
+## Remote power control using ipmitool
+
+To remote control supermicro you can use ipmitool.
+
+```bash
+ipmitool -H [bmc_ip] -U [bmc_username] -P [bmc_password] chassis power off
```

Comment:
Where should I look for credentials?

Reply:
Could be a good idea to store them in snipeit, along with the other information, such as IP addresses?

Reply:
@krystian-hebel I added credentiale on snipeit